app-admin/sudo/sudo-1.9.8_p2-r1.ebuild

```diff
@@ -200,8 +202,7 @@ src_install() {
 		doins "${T}"/ldap.conf.sudo
 		fperms 0440 /etc/ldap.conf.sudo
 
-		insinto /etc/openldap/schema
-		newins doc/schema.OpenLDAP sudo.schema
+		#Flatcar: we don't ship OpenLDAP schemas
```

Comment:
I try to avoid removing lines if I can just comment them out.

Reply:
Sure. Will just comment it out here.